### DISCUSSION TITLE
How do I tag a discussion?

### DISCUSSION THREAD

**Discussion by sandeepchads** [community/community#3570] [Discussions]
I would love to tag a discussion and then be able to search / filter it using those tags?

**Comment by shaunagm** [comment]
Agreed - the lack of tags/labels is the main reason I'm not switching discussions out of the issue tracker.

**Reply by marcastel** [reply]
Agreed. Labels are fine, but, since we have normalised them to categorise and organise issues, they are hardly pertinent for discussions. One could have a separate list of labels for discussions and issues, but this gets messy to administrate and use. Tags allow topical, possibly user-driven, classification of discussion topics.
